web/app/components/comment/comment.jsx

```diff
@@ -306,6 +307,8 @@ export default class Comment extends Component {
             <span className={b('comment__score', {}, { view: o.score.view })}>
               <span
                 className={b('comment__vote', {}, { type: 'up', selected: scoreIncreased, disabled: isGuest || isCurrentUser })}
+                role="button"
+                aria-disabled={isGuest || isCurrentUser ? 'true' : 'false'}
```

Comment:
You can use just `isGuest || isCurrentUser` as a value here, can't you?

Reply:
I thought about it. However, in all specifications I always saw `aria-something="true"` or `"false"` (note the quotes). So would it be OK to leave it as you suggest? If so, I'll be glad to remove those unnecessary boolean values :)